packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,285 @@
+  }
+
+  :hover,
+  :focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledLiveIndicator = styled.span`
+  color: ${C_POSTBOX};
+  text-transform: uppercase;
+  text-decoration: none;
+  :hover {
+    text-decoration: none;
+  }
+`;
+
+const BulletinBodyWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+`;
+
+const BulletinImageWrapper = styled.div`
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledBulletinImage = styled.img`
+  display: block;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+`;
+
+const BulletinBody = ({
+  title,
+  summary,
+  type,
+  liveText,
+  isLive,
+  script,
+  service,
+  link,
+  ctaText,
+  hiddenTitleText,
+  dir,
+}) => (
+  <BulletinBodyWrapper dir={dir}>
+    <StyledHeading script={script} service={service}>
+      <VisuallyHiddenText>{hiddenTitleText}</VisuallyHiddenText>
+      {isLive && <LiveIndicator text={liveText} />}
+      <HeadingLink href={link} dir={dir}>
+        {title}
+      </HeadingLink>
+    </StyledHeading>
+    <StyledSummary script={script} service={service}>
+      {summary}
+    </StyledSummary>
+    <PlayCTA type={type} isLive={isLive} dir={dir}>
+      <IconWrapper>{mediaIcons[type.toLowerCase()]}</IconWrapper>
+      {ctaText}
+    </PlayCTA>
+  </BulletinBodyWrapper>
+);
+
+BulletinBody.propTypes = {
+  title: string.isRequired,
+  summary: string.isRequired,
+  type: oneOf(['video', 'audio']).isRequired,
+  liveText: string,
+  script: shape(scriptPropType).isRequired,
+  service: string.isRequired,
+  link: string.isRequired,
+  ctaText: string.isRequired,
+  hiddenTitleText: string.isRequired,
+  dir: oneOf(['ltr', 'rtl']).isRequired,
+  isLive: bool.isRequired,
+};
+
+BulletinBody.defaultProps = {
+  liveText: null,
+};
+
+const LiveIndicator = ({ text }) => (
+  <StyledLiveIndicator>{text}</StyledLiveIndicator>
+);
+
+LiveIndicator.propTypes = {
+  text: string.isRequired,
+};
+
+export const RadioBulletin = ({
+  title,
+  summary,
+  liveText,
+  dir,
+  script,
+  service,
+  link,
+  ctaText,
+  hiddenTitleText,
+  isLive,
+}) => (
+  <BulletinWrapper dir={dir}>
+    <BulletinBody
+      title={title}
+      summary={summary}
+      type="audio"
+      liveText={liveText}
+      script={script}
+      service={service}
+      link={link}
+      ctaText={ctaText}
+      hiddenTitleText={hiddenTitleText}
+      dir={dir}
+      isLive={isLive}
+    />
+  </BulletinWrapper>
+);
+
+RadioBulletin.propTypes = {
+  title: string.isRequired,
+  summary: string.isRequired,
+  liveText: string,
+  dir: oneOf(['rtl', 'ltr']).isRequired,
+  script: shape(scriptPropType).isRequired,
+  service: string.isRequired,
+  link: string.isRequired,
+  ctaText: string.isRequired,
+  hiddenTitleText: string.isRequired,
+  isLive: bool,
+};
+
+RadioBulletin.defaultProps = {
+  liveText: null,
+  isLive: false,
+};
+
+export const BulletinImage = ({ src, alt }) => (
+  <BulletinImageWrapper>
+    <StyledBulletinImage src={src} alt={alt} />
+  </BulletinImageWrapper>
+);
```

Comment:
Based on looking at the `StoryPromo` component in Simorgh, this may have to change before actually being used to work in a similar way so that amp pages work properly.